src/components/course/repo/helper.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Converts an mongoose update many result to an internal TO
+ * @param {*} param0
+ * @param {1|0} param0.ok 0 for error
+ * @param {Integer} param0.n Number of elements matched the given filter
+ * @param {Integer} param0.nModified Number of elements updated
+ */
+const updateManyResult = ({ ok, n, nModified }) => {
+	if (ok !== 1) {
+		// TODO warn/throw
```

Comment:
Maybe adding warning that we not it in one way that we can remove the TODO.

Reply:
add error log here